roles/neutron-data/tasks/main.yml

```diff
@@ -49,6 +49,18 @@
     - ursula_os == 'rhel'
     - neutron.plugin == 'ml2'
 
+- name: mask unwanted neutron services on compute (rhel)
+  command: systemctl mask {{ item.name }}
+  with_items:
+    - "{{ neutron.services.neutron_server }}"
+    - "{{ neutron.services.neutron_dhcp_agent }}"
+    - "{{ neutron.services.neutron_l3_agent }}"
+    - "{{ neutron.services.neutron_metadata_agent }}"
+  when:
+    - ursula_os == 'rhel'
+    - inventory_hostname in groups['compute']
```

Comment:
Because `neutron-data` role only runs for `compute:network` hosts, we could get rid of this check but do we want to rely on that and not be strict at checking?